examples/todoapp/test/feature_statistics/view/statistics_golden_test.dart

```diff
@@ -1,18 +1,25 @@
 import '../../helpers/golden_helper.dart';
-import '../../helpers/models/scenario.dart';
 import '../../stubs.dart';
 import '../factory/statistics_factory.dart';
 
 void main() {
   runGoldenTests([
-    generateDeviceBuilder(
-        widget: statisticsFactory(todosStats: Stubs.todoListStatisticsEmpty),
-        scenario: Scenario(name: 'statistics_empty')),
-    generateDeviceBuilder(
-        widget: statisticsFactory(todosStats: Stubs.todoListStatistics),
-        scenario: Scenario(name: 'statistics_success')),
-    generateDeviceBuilder(
-        widget: statisticsFactory(),
-        scenario: Scenario(name: 'statistics_loading')),
+    buildScenario(
+      scenario: 'statistics_empty',
+      widget: statisticsFactory(
+        todosStats: Stubs.todoListStatisticsEmpty,
+      ),
+    ),
+    buildScenario(
+      scenario: 'statistics_success',
+      widget: statisticsFactory(
+        todosStats: Stubs.todoListStatistics,
+      ),
+    ),
+    buildScenario(
+      scenario: 'statistics_loading',
+      customPumpBeforeTest: animationCustomPump,
+      widget: statisticsFactory(),
```

Comment:
Do we still need the `animationCustomPump` ? If not, please remove this global function.

Reply:
Removed! 👍🏼